packages/governance/src/types-ambient.js

```diff
@@ -574,11 +574,11 @@
  * @property {VoteOnParamChanges} voteOnParamChanges
  * @property {VoteOnApiInvocation} voteOnApiInvocation
  * @property {VoteOnOfferFilter} voteOnOfferFilter
- * @property {() => Promise<LimitedCreatorFacet<CF>>} getCreatorFacet - creator
+ * @property {() => ERef<LimitedCreatorFacet<CF>>} getCreatorFacet - creator
```

Comment:
hm... we typically use `ERef` for parameter types and `Promise` for return types. I suppose this serves as both, in a way? Is this covariant or contravariant? (_feel free to treat that as rhetorical. My head is spinning._)

Reply:
ERef is our most concise way to say, https://github.com/endojs/endo/blob/4c8702a77923c4cf757718296d34d933b30b63a4/packages/eventual-send/src/index.d.ts#L29-L30

So the change here is just to allow `getCreatorFacet` to return the facet (not always a promise for it)